src/Uno.UI/UI/Xaml/Controls/ContentControl/ContentControl.cs

```diff
@@ -317,6 +317,20 @@ private protected override void OnLoaded()
 			}
 		}
 
+		protected override void OnPointerPressed(PointerRoutedEventArgs args)
+		{
+			if (args.Handled)
+			{
+				base.OnPointerPressed(args);
+				return;
+			}
+
+			if (args.GetCurrentPoint(this).Properties.IsLeftButtonPressed)
+			{
+				args.Handled = Focus(FocusState.Pointer);
+			}
```

Comment:
Did you confirm that on Windows `ContentControl` flags the event has handled. This might have significant side effects, I would be surprised we did detect that behavior earlier.

Also @MartinZikmund might have an opinion about the focus management :)

Reply:
Upon further testing, it's at the ScrollViewer level, not ContentControl level, that the event gets handled.